### ISSUE TITLE
how can i use electron-settings in angular2?

### ISSUE THREAD

**Issue by luimucar** [nathanbuchar/electron-settings#89]
i am trying to import in an angular2 app but it does not work? 

import { Settings} from 'electron-settings';

Any help?

**Comment by nathanbuchar** [comment]
I'd be happy to help but unfortunately I'm unfamiliar with Angular. When you say that it "does not work", what do you mean exactly? Are you getting a node error?

**Comment by luimucar** [comment]
Thank you nathan,

my scenario is the following: I have an angular2 app working properly in the browser. We've decided to make it work inside electron. in the process, we want the user to set an external config file. this is when I found electron-settings. 

I found the problem. it was related with webpack. I just needed to include a parameter, target='electron-renderer'. In that way, when webpack builds the app for delivery, it includes necessary electron libraries (used by electron-settings as well) fs, and so on. so problem solved. :)
